src/assets/icons/index.ts

```diff
@@ -13,5 +13,26 @@ export { ReactComponent as PlusXL } from './PlusXL.svg';
 export { ReactComponent as Share } from './Share.svg';
 export { ReactComponent as Star } from './Star.svg';
 export { ReactComponent as Calendar } from './Calendar.svg';
-import * as SocialIcons from './Social';
-export { SocialIcons };
```

Comment:
@sivanharel You can't remove this because it's a breaking change

Reply:
Ok I will revert it

Reply:
I'd suggest adding a comment on top of it that this should be deprecated